h2o-py/h2o/estimators/word2vec.py

```diff
@@ -203,3 +203,10 @@ def max_runtime_secs(self, max_runtime_secs):
         self._parms["max_runtime_secs"] = max_runtime_secs
 
 
+
+    def import_pretrained(self):
```

Comment:
I think this API is confusing because it lets you call it even when `pre_trained` frame is not specified. It doesn't allow for any checks.

What I had it mind was a static method that would let you specify only the relevant parameters:

    w2v_model = H2OWord2vecEstimator.from_external(pre_trained=embeddings)

Reply:
The from_external will be used